Source/BundleHelper.swift

```diff
@@ -0,0 +1,89 @@
+//
+//
+//
+
+import UIKit
+
+public class BundleHelper {
+
+    private let bundle: Bundle
+
+    /// Instantiate bundle for the specified class and bundle name.
+    /// - parameter targetClass: the class name in the bundle it is in.
+    /// - parameter bundleName: bundle name with resources.
```

Comment:
Instantiates bundle for the specified class and bundle name.
The class with which the bundle is associated
The resource bundle name, used as fallback